components/FormManual/UploadButton.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import Upload from "@/public/icons/Upload.svg";
+
+export default function UploadButton({ uploadImage }) {
+  return (
+    <>
+      <StyledImageInput>
+        <IconTextWrapper>
+          <Upload />
+          Upload Image
+        </IconTextWrapper>
+      </StyledImageInput>
+      <HiddenImageInput
+        name="image"
+        type="file"
+        accept="image/*"
+        id="image"
+        onChange={(event) => uploadImage(event.target.files[0])}
+      />
```

Comment:
you seem to have two inputs for the file. can't you just use one, the styleImageInput and add the id, accpt, name type and onChange to that?

Reply:
Because I thought I needed the hidden one as the real upload button and since you can't style it I put the other styled one on top.